README.md

```diff
@@ -0,0 +1,223 @@
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+https://user-images.githubusercontent.com/68339870/226114036-d7a98d0a-f829-4c19-90d4-cef3ec609c75.mp4
+
+
+
+<!-- - [Live Demo Link]() -->
+[demo](https://develop--zesty-mooncake-1a3672.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Code Editor
+- Git installed
+- Node js installed
+
+You need to have Node.js installed.
+
+### Setup
+
+
+Clone this repo into your desire folder and then cd into it.
+https://github.com/amklass11/react-capsyone.git
+
+### Install
+
+sh
+npm install
+
+
+### Usage
+
+Run following command to start development server with live update.
+
+sh
+npm run start
+
+
+To build the app
+
+sh
+npm run build
+
+
+### Run tests
+
+There is no tests in this project but if you want to check linter errors run following command.
+
+sh
+npm run linters
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Amk Lass**
+
+
+- GitHub:[@amklass11](https://github.com/amklass11)
+
+- LinkedIn:[@AmkLass](https://www.linkedin.com/in/amk-lass-521565196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- [ ] *[Add drag and drop function]*
+- [ ] *[Update UI/UX Design]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please don't forget to follow me and give me advice so that I can improve more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+<!-- > Add at least 2 questions new developers would ask when they decide to use your project. -->
+
+- *Can you make a list of 10 Most Popular Books based of the inputs in your page?*
+
+  - Hi! Thank you for the suggestion. It would be an honor to please our readers. We will do that ASAP.
+
+<!-- - *[Question_2]*
+
+  - [Answer_2] -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix the license link as the previous reviewers mentioned because It's still broken.

![image](https://user-images.githubusercontent.com/68796232/227718593-46bdc8f8-7599-4f8e-900b-6fd27845a8b1.png)